crates/bevy_app/src/plugin_registry.rs

```diff
@@ -0,0 +1,470 @@
+use bevy_utils::HashMap;
+use std::panic::{catch_unwind, resume_unwind, AssertUnwindSafe};
+
+use crate::Plugin;
+use crate::{App, PluginState};
+
+/// Plugin registry state in the application
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, PartialOrd, Ord)]
+pub enum PluginRegistryState {
+    #[default]
+    /// No plugin has been added.
+    Idle,
+    /// Plugins are initialized.
```

Comment:
The wording should be unambiguously either present tense or past tense. Right now, these doc comments aren't clear between "Plugins are being initialized" and "Plugins have been initialized".